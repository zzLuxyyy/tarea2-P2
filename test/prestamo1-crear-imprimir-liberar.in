crearSocio 12345678 Jim Dear 10 03 2004
crearFecha 01/01/1966
crearLibro 411222335 "Todos los fuegos el fuego" "Julio" "Cortázar" "Ocho cuentos que combinan lo cotidiano con lo fantástico" 1
crearFecha 08/09/2024
crearPrestamo
imprimirSocioPrestamo
imprimirLibroPrestamo
imprimirFechaRetiroPrestamo
imprimirPrestamo
liberarPrestamo
Fin