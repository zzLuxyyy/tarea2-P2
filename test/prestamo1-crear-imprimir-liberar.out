1>Socio de ci 12345678 creado exitosamente.
2>3>4>5>Préstamo de libro de isbn 411222335 a socio 12345678 creado exitosamente.
6>Socio del préstamo:
Socio 12345678: Jim Dear
Fecha de alta: 10/3/2004
Géneros favoritos: 
7>Libro del préstamo:
Libro 411222335
Título: Todos los fuegos el fuego
Autor: Julio Cortázar
Descripción: Ocho cuentos que combinan lo cotidiano con lo fantástico
Género: 1
Fecha de edición: 1/1/1966
8>Fecha de retiro del préstamo:
8/9/2024
9>Préstamo de libro Todos los fuegos el fuego a Jim Dear.
8/9/2024
No retornado
10>Se ejecutó 'liberarTPrestamo' exitosamente.
11>Fin.
